README.md

```diff
@@ -16,7 +16,7 @@ Access design documentation for features coming in future releases, post OpenShi
 - [YAML Editor Updates](http://openshift.github.io/openshift-origin-design/web-console/future-openshift/code-editor-updates/code-editor-updates)
 
 ## Developer Perspective
-Access future design documentation specific to the developer perspective of OpenShift.
+[Access future design documentation specific to the developer perspective of OpenShift.](http://http://openshift.github.io/openshift-origin-design/web-console/developer/developer)
```

Comment:
Should this work at the moment? Or not until this is all merged?

Reply:
After this gets merged (and with the suggested change below) it'll link to a website-based version of [this file](https://github.com/openshift/openshift-origin-design/blob/567ea78ffaaee0ec5ecfec1636356539efc2a386/web-console/developer/developer.md).

Reply:
Ah okay cool, thanks @andybraren !!